src/main/java/frc/robot/AutoCenter.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot;
+
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class AutoCenter extends Command {
+  
+  boolean end = false;
+  double kP = -0.05;
+  private final Timer timer = new Timer();
+  private final CommandSwerveDrivetrain driveSubsystem;
+  private final SwerveRequest.FieldCentric drive;
+  
+  /** Creates a new AutoCenter. */
+  public AutoCenter(CommandSwerveDrivetrain driveSubsystem, SwerveRequest.FieldCentric drive) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.driveSubsystem = driveSubsystem;
+    this.drive = drive;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    timer.start();
+    end = false;
+    if (LimelightHelpers.getTX("") == 0) {
```

Comment:
The requirement that TX be 0 seems kinda tight here? How often does it actually return 0 and is perfectly aligned with the target?